docs/source/traditional-ml/serving-multiple-models-with-pyfunc/index.rst

```diff
@@ -0,0 +1,52 @@
+Serving Multiple Models on a Single Endpoint with Custom PyFunc
+===============================================================
+
+This tutorial addresses a common scenario in machine learning: serving multiple models through a 
+single endpoint. Utilizing services like Sagemaker's Multi-Model Endpoints, you can host numerous 
+models under one endpoint, simplifying deployment and cutting costs. We'll replicate this 
+functionality using any MLflow-compatible service combined with a custom PyFunc implementation.
+
+Here are some reasons to consider this design:
+
+- **Simplified Inference**: We will maintain a single model endpoint instead of one per model, which drammatically reduces maintanence and providioning complexity. 
+- **Reduced Serving Cost**: Endpoints cost money! If your hosting service charges for compute and not memeory, this will save you money.
+
+0 - What's in this tutorial?
+----------------------------
+This guide walks you through the steps to serve multiple models from a single endpoint, breaking 
+down the process into:
+
+1. Create many demo sklearn models, each trained on data corresponding to a single day of the week.
+2. Wrap those models in a custom PyFunc model to support multi-model inference.
+3. Perform inference on the custom PyFunc model.
+4. Locally serve the custom PyFunc model and query our endpoint.
+
+After completing this tutorial, you'll be equipped to efficiently serve multiple models from a 
+single endpoint.
+
+1 - What is PyFunc?
+-------------------
+Custom PyFunc is a powerful MLflow feature that lets users cusomize model behavior. Going forward we
+assume basic working knowledge of PyFunc, so if you're unfamiliar, check out the 
+``Creating Custom PyFunc`` tutorial.
```

Comment:
Should we add a link to ``Creating Custom PyFunc``? BTW I'm a little bit confused by what PyFunc means here, does it refer to PyFunc models or pyfunc flavor in general?

Reply:
My understanding was that `custom PyFunc` is a term that encompasses the implementation of a child of a `mlflow.pyfunc.PythonModel` as outlined in the [tutorial here](https://mlflow.org/docs/latest/traditional-ml/creating-custom-pyfunc/part2-pyfunc-components.html). Is that correct? And any suggestions on making this more clear?

Reply:
I would prefer using custom Pyfunc models instead, as all over the end it's creating a model, pyfunc is more like a flavor 🤔